app/views/events/_material_area.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="container" id="event_material_area">
```

Comment:
Checking this again, using `.container` is not the proper way using bootstrap since this template is already loaded in a `.container` block element.
![bildschirmfoto 2017-01-05 um 21 40 36](https://cloud.githubusercontent.com/assets/6578648/21696756/aa83f446-d38f-11e6-8d83-f9ee3982e28b.png)

Moreover it could be better using a block element like this for wrapping your table: `<div class="col-xs-12 col-sm-12 col-md-10 col-lg-10 col-xs-offset-0 col-sm-offset-0 col-md-offset-1 col-lg-offset-1"></div>`. This would result into a narrower table, since it only has two columns.